openslides_backend/presenter/number_of_users.py

```diff
@@ -32,14 +31,14 @@ class NumberOfUsers(BasePresenter):
 
     def get_result(self) -> Any:
         organization = self.datastore.get(
-            FullQualifiedId(Collection("organization"), ONE_ORGANIZATION),
+            cast(FullQualifiedId, "organization/1"),
```

Comment:
there seem to be more places like this, not gonna comment them all :D

Reply:
Another idea: use another constant `ONE_ORGANIZATION_FQID` to avoid reconstructing it everytime.